python/dgl/contrib/cugraph/cugraph_storage.py

```diff
@@ -0,0 +1,220 @@
+# Copyright (c) 2019-2022, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: Requires cuGraph nightly cugraph-22.06.00a220417 or later
+
+import cugraph
+from cugraph.experimental import PropertyGraph
+from .cugraph_utils import cugraphToDGL
+import dgl
+import cupy
+import torch
+
+
+class CuGraphStorage(object):
```

Comment:
we shouldn't need to inherit from `object` for the versions of python supported by cugraph.

Reply:
fix, will be in next push